<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QDialog" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Usuário:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nome">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomeuser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Função</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>741</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Caixa</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_cadastrocliente">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Cadastro de Cliente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_venda">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Venda</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pedidos">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Pedidos</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Admin</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cadastro de Usuário</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_configloja">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Configuração da Loja</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cadastroendereco">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cadastro de Endereço</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_analisevendas">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Análise de Vendas</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Configurar Cardápio</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_cadastrarmenu">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Cadastrar Menus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_precoitem">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cadastrar Preço de Item</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_produto">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cadastrar Produto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_montarcardapio">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Montar Cardápio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_submenu">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SubMenu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ligar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Ligar/Desligar itens</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Entrega</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_entrega">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Aŕea de Entrega</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_condominio">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Condomínios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_precosentrega">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Preços de Entrega</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
